clifford/numba/_multivector.py

```diff
@@ -53,19 +49,29 @@ def value_type(self):
 
 @property
 def _numba_type_(self):
+    # If the array is not 1D we can't do anything with it
+    if self.value.ndim != 1:
+        return None
+
     layout_type = self.layout._numba_type_
 
-    cache = layout_type._cache
+    relevant_cache = layout_type._c_cache
     dt = self.value.dtype
 
+    if not self.value.flags.c_contiguous:
+        relevant_cache = layout_type._a_cache
```

Comment:
I'd have a slightly easier time reading this as:
```suggestion
    dt = self.value.dtype
    if self.value.flags.c_contiguous:
        relevant_cache = layout_type._c_cache
    else:
        relevant_cache = layout_type._a_cache
```